### DISCUSSION TITLE
Can't find /etc/nsm/securityonion.conf in the latest version

### DISCUSSION THREAD

**Discussion by ziko0079** [Security-Onion-Solutions/securityonion#2553] [Unsupported Versions] [answered]
Can't find /etc/nsm/securityonion.conf in the latest version in order to manage snapshots as it's indicated in the docs 
https://docs.securityonion.net/en/2.3/elasticsearch.html?highlight=securityonion.conf#snapshots

**Comment by dougburks** [comment, marked as answer]
There is no securityonion.conf in Security Onion 2. That documentation was left over from the previous version. I've just removed it.

**Reply by ziko0079** [reply]
so how can we map the container volume to the host directory for snapshot repository (so that we can add ELASTICSEARCH_OPTIONS="-v /backups:/backups") ?
thank you in advance

**Reply by weslambert** [reply]
You can copy `/opt/so/saltstack/default/salt/elasticsearch/init.sls` to `/opt/so/saltstack/local/salt/elasticsearch/init.sls` and modify it there.
